pmm-admin.go

```diff
@@ -282,11 +283,14 @@ a new user 'pmm@' automatically using the given (auto-detected) MySQL credential
 Table statistics is automatically disabled when there are more than 10000 tables on MySQL.
 
 [name] is an optional argument, by default it is set to the client name of this PMM client.
+[-- exporter_args] are additional exporter arguments.
 		`,
 		Example: `  pmm-admin add mysql:metrics --password abc123
   pmm-admin add mysql:metrics --password abc123 --create-user
   pmm-admin add mysql:metrics --password abc123 --port 3307 instance3307
-  pmm-admin add mysql:metrics --user rdsuser --password abc123 --host my-rds.1234567890.us-east-1.rds.amazonaws.com my-rds`,
+  pmm-admin add mysql:metrics --user rdsuser --password abc123 --host my-rds.1234567890.us-east-1.rds.amazonaws.com my-rds
+  pmm-admin add mysql:metrics -- --collect.perf_schema.eventsstatements
+  pmm-admin add mysql:metrics -- --collect.perf_schema.eventswaits=false`,
```

Comment:
Examples taken from: https://www.percona.com/doc/percona-monitoring-and-management/pmm-admin.html#passing-parameters-to-a-monitoring-service